lib/meadow/data/preservation_check_writer.ex

```diff
@@ -17,7 +17,7 @@ defmodule Meadow.Data.PreservationCheckWriter do
     "file_set_role",
     "sha256",
     "sha1",
-    "preservation_location",
+    "preservation_exists",
```

Comment:
This looks like the wrong field? This will result in `preservation_exists` and `exists_in_preservation` instead of `preservation_location` and `preservation_exists`.